### ISSUE TITLE
Add code snippet to docs

### ISSUE THREAD

**Issue by ErikSchierboom** [exercism/csharp#369]
In Exercism V2, each track's page shows a small code sample. It default to the first 10 lines of hello world, but in our case this doesn't look very pretty: https://v2.exercism.io/tracks/csharp

I think we should provide a docs/SNIPPET.txt file with a slightly better formatted version of the hello world exercise.

**Comment by jpreese** [comment]
Did we just want to update the hello world example? I always thought it could be slightly confusing for new comers as it uses a lambda expression right out of the gate.

**Comment by ErikSchierboom** [comment]
@jpreese The example shouldn't be confusing for newcomers as they should never even see the example. The purpose of the example file is to prove that the exercise can be solved (and also to learn about the exercise in relation to the track). So that shouldn't be a problem. However, it might be the easiest solution to just update the example with a better formatted version.

**Comment by robkeim** [comment]
@ErikSchierboom I agree that this snippit definitely doesn't look good, but what are the guidelines for the code sample? If we're looking to attract people to the track maybe we should have a very simple sample removing as much scaffolding as possible which makes C# look easy. Something like this:
```
class HelloWorld
{
    string Hello()
    {
        return "Hello, world!";
    }
}
```

Obviously that's not as "cool" as the lamda solution so I think it depends on what we're aiming for with the code snippit.

**Comment by ErikSchierboom** [comment]
I love it! Simple and clean, I think it will appeal to newcomers.

**Comment by ErikSchierboom** [comment]
Closed by #376
